etl/scripts/faostat/archive_old_datasets.py

```diff
@@ -0,0 +1,267 @@
+"""Archive unused grapher FAOSTAT datasets and etl steps.
+
+TODO: Most of the logic in this script could (and eventually should) be integrated with VersionTracker.
+
+"""
+
+import argparse
+from typing import List
+
+import pandas as pd
+from MySQLdb.connections import Connection
+from structlog import get_logger
+
+from etl import db
+from etl.helpers import VersionTracker
+from etl.scripts.faostat.create_new_steps import find_latest_version_for_step
+from etl.scripts.faostat.shared import NAMESPACE
+
+log = get_logger()
+
+INTERACTIVE = True
+
+
+def list_active_db_datasets(db_conn: Connection) -> pd.DataFrame:
+    query = """
+    SELECT d.id, d.name, d.namespace, d.version, d.shortName
+    FROM datasets d
+    JOIN variables v
+    ON v.datasetId = d.id
+    JOIN chart_dimensions cd
+    ON v.id = cd.variableId
+    WHERE d.isArchived IS FALSE
```

Comment:
maybe not relevant but: do we want to check if the dataset is private/public here?

Reply:
I think I prefer to be open, just in case a private dataset is using any of the relevant variables.